css/cssom/selectorSerialize.html

```diff
@@ -116,6 +116,26 @@
             test(function() {
                 assert_selector_serializes_to(escaped_ns_rule + "[ns\\:odd|odd\\:name]", "[ns\\:odd|odd\\:name]");
             }, "escaped character in both attribute prefix and name");
+
+            test(() => {
+              assert_selector_serializes_to("\\\\", "\\\\");
```

Comment:
It's arguable that this and many other things (existing and new) in this test would be more readable as `` String.raw`\\` `` rather than as `"\\\\"`, but I think it's probably OK to keep this as-is.